service/init.go

```diff
@@ -351,13 +350,16 @@ func (smf *SMF) Start() {
 	udp.Run(pfcp.Dispatch)
 
 	for _, upf := range context.SMF_Self().UserPlaneInformation.UPFs {
-		if upf.NodeID.NodeIdType == pfcpType.NodeIdTypeFqdn {
+		if upf.NodeID.NodeIdType == context.NodeIdTypeFqdn {
```

Comment:
I am not sure I understand this change. Given that you are using `go-pfcp`, should this be as follows?
```suggestion
		if upf.NodeID.NodeIdType == ie.NodeIDFQDN {
```
Of course, it is required to import `"github.com/wmnsk/go-pfcp/ie"`

In general, I see that you are creating several structs that already exist in `go-pfcp`. Should not we use the structs from `go-pfcp`? or am I overlooking something?

Reply:
The intent is to only use go-pfcp in code responsible for sending or receiving PFCP messages, not to use as structs all throughout the code. 

In the future I would like to simplify a bunch of those structs as we don't really need most of their content. In this PR I really just wanted to make the switch of modules without any other refactoring as it's already a large change.